Ednpoints.txt

```diff
@@ -0,0 +1,132 @@
+The same routes for users, chefs and delivery
+POST: /auth/register - registration
+POST: /auth/login - enter to account
+GET: /auth/logout - exit from account
+POST: /auth/forgot-password - request to change password
+POST: /auth/reset-password - password reset
+
+/************/
+
+USER ENDPOINTS:
+
+GET: /users/:userId - get user's info
```

Comment:
should we have endpoint for retrieving all the users (for admin puurpooses, for ex)?

Reply:
Yes, that's right, in the admin section we added an endpoint to get a list of all users